pages/apim/3.x/how-tos/how-to-enable-opentracing-with-jaeger.adoc

```diff
@@ -1,102 +1,104 @@
+
+```
+docker run -d --name jaeger \
+  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
+  -p 5775:5775/udp \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 14268:14268 \
+  -p 14250:14250 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:1.27
+```
+NOTE: For the most recent documentation on starting the Jaeger Docker image, see https://www.jaegertracing.io/[Jaegertracing.io^].
+
+*PRO-TIP:* By running this Docker command, you'll also get access to the JaegerUI.
+
+=== 3. Confirm Jaeger Is Enabled On Docker Desktop
+You can confirm Jaeger has been enabled and is running in Docker Desktop.
+
+image:apim/3.x/how-tos/enable-opentracing/jaeger-docker-desktop.png[Jaeger is running on Docker Desktop].
 
 === 4. Install The .ZIP File For The Jaeger Tracer Plugin
-Since the Jaeger tracer is not bundled by default, 
-link:/https://download.gravitee.io/#graviteeio-apim/plugins/tracers/gravitee-tracer-jaeger/[click here] and install the plugin for the Gravitee Jaeger Tracer .ZIP file on your Docker instance. 
-
-=== 5. Test OpenTracing 
-
-* 1. Now that you have OpenTracing enabled, run your API Gateway and APIM. 
-
-* 2. Open APIM and choose an API that already has a policy assigned to it (or create a test API and add any policies you like for this test). Now, call your API. 
-
-* 3. To see your calls, open the helpful, user-friendly JaegerUI by visiting http://localhost:16686 (note that JaegerUI was automatically installed earlier in the process). Click *Search* and find the API you called. 
-
-image:apim/3.x/how-tos/enable-opentracing/tracing-info-general.png[General tracing info from JaegerUI.] 
-
-In these examples, you’ll also notice that each policy traversed is listed on a different line to provide greater visibility and assist with debugging. You can even see timestamp info on the detailed view. 
-
-image:apim/3.x/how-tos/enable-opentracing/tracing-info-detailed.png[Detailed tracing info from JaegerUI.] 
-
-
-== Configure Environment Variables 
-
-You can also configure this with environment variables in the Jaeger Gateway. Instead of using the `gravitee.yml`, you can use this as a base and add the environment variables in the gateway itself (see below). 
-
-image:apim/3.x/how-tos/enable-opentracing/environment-variables-large.png[Environment variables in Jaeger Gateway.] 
-
-Here, you can change the variables as needed. 
-
-image:apim/3.x/how-tos/enable-opentracing/environment-variables-closeup.png[Environment variables in Jaeger Gateway, fig. 2.] 
-
-
-== How To Enable OpenTracing Via Docker Compose 
-
-Enabling OpenTracing is easy -- get started using Jaegar as a tracer with this Docker Compose. 
+Since the Jaeger tracer is not bundled with APIM by default,
+link:https://download.gravitee.io/#graviteeio-apim/plugins/tracers/gravitee-tracer-jaeger/[click here^] to download the Gravitee Jaeger Tracer .ZIP file, which adds the tracer to the configuration of your APIM Gateway instance.
+// more info needed
```

Comment:
we have more info with the docker-compose version

for this part, it's related to the classical installation of plugins http://localhost:4000/apim/3.x/apim_installguide_gateway_configuration.html#configure_the_plugins_repository